src/decoding.ts

```diff
@@ -0,0 +1,79 @@
+import { Protobuf, Reader } from "as-proto";
+
+export function decodeMsgDelegate(a: Uint8Array): MsgDelegate {
+    return Protobuf.decode<MsgDelegate>(a, MsgDelegate.decode);
+}
```

Comment:
Looks like the formatting is different in this file. Let's use 2 spaces for indentation everywhere.